### PR TITLE
move folders provide docs

### PR DESCRIPTION
**Pull request by ScottSuarez**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -46,4 +46,11 @@ validator:
 		bundle; \
 		bundle exec compiler -e terraform -f validator -o $(OUTPUT_PATH) $(mmv1_compile);
 
+serialize:
+	cd tpgtools;\
+		go run . --path "api" --overrides "overrides" --mode "serialization" > temp.serial;\
+		cat temp.serial > serialization.go;\
```
Comment (slevenick) [inline GNUmakefile:52]: `mv temp.serial serialization.go`?

Or `> serialization.go`?

File: tpgtools/main.go [hunk 2]
```diff
@@ -257,7 +256,7 @@ func loadDocument(packagePath string, f *os.FileInfo) *openapi.Document {
 	}
 	err = yaml.Unmarshal(b, document)
 	if err != nil {
-		glog.Exit(err)
+		panic(err)
```
Comment (slevenick) [inline tpgtools/main.go:259]: `glog.Exit` is generally better than panic I believe? Why would we change this

File: GNUmakefile (hunk 1 above)
Comment (ScottSuarez) [reply GNUmakefile:52]: Can't do `> serialization.go` code bugs out. Need an intermediate step.

File: tpgtools/main.go (hunk 2 above)
Comment (ScottSuarez) [reply tpgtools/main.go:259]: oh this a debugging artifact, will remove

File: tpgtools/README.md [hunk 3]
```diff
@@ -79,3 +79,100 @@ For example, override entries will look like the following:
...
+To do this
+```
+cd tpgtools
+go get -u github.com/GoogleCloudPlatform/declarative-resource-client-library
+make serialize
+```
+
+To do so, first create a folder in the relevant product
+```
+$ cd overrides/{{product}}/
```
Comment (slevenick) [inline tpgtools/README.md:98]: This is now in a different place, `samples/{{product}}` right?

Comment (ScottSuarez) [reply tpgtools/README.md:98]: I am moving `samples` to `overrides` with this pr

File: tpgtools/README.md [hunk 4]
```diff
@@ -79,3 +79,100 @@ For example, override entries will look like the following:
...
+
+If you need to hide sample from doc or hide a sample from docs you can do so here as well.
+```
+doc_hide:
+  - basic.tft
+test_hide:
+  - basic_trigger.yaml
+```
+
+Any files with a `.tft` (terraform template) extension located in the `override/{{product}}samples/{{resource}}` directory
```
[the remark this answers is not in the document]
Comment (ScottSuarez) [reply tpgtools/README.md]: done

File: tpgtools/main.go (hunk 2 above)
Comment (ScottSuarez) [reply tpgtools/main.go:259]: done

File: GNUmakefile (hunk 1 above)
Comment (ScottSuarez) [reply GNUmakefile]: done

**Review by slevenick — APPROVED** [review]

**Pull request merged**